package/files/update_rmt_app_dir_permissions.sh

```diff
@@ -21,12 +21,5 @@ fi
 secret_key_files=('config/secrets.yml.key' 'config/secrets.yml.enc')
 
 for secretFile in ${secret_key_files[@]}; do
-  file_path="$app_dir/$secretFile"
-  if [[ -e $file_path ]]; then
-    if [[ "$(stat -c "%U %G" $file_path)" == "root root" ]]; then
-      chmod 0640 $file_path
-      chown -h root:nginx $file_path
-    fi
-  fi
-
+  [[ -e $file_path ]] && rm -f "$app_dir/$secretFile"
```

Comment:
```suggestion
  file_path="$app_dir/$secretFile"
  [[ -e $file_path ]] && rm -f "$app_dir/$secretFile"
```
otherwise:
> [   64s] /usr/lib/rmt/update_rmt_app_dir_permissions.sh: line 24: file_path: unbound variable

Reply:
right, got over eager here hehe. should be:

```bash
  file_path="$app_dir/$secretFile"
  [[ -e $file_path ]] && rm -f "$file_path"
```

Reply:
fixed in 7a28d1d6